spacy/errors.py

```diff
@@ -871,6 +871,8 @@ class Errors:
              "the documentation:\nhttps://spacy.io/usage/models")
     E1020 = ("No `epoch_resume` value specified and could not infer one from "
              "filename. Specify an epoch to resume from.")
+    E1021 = ("`pos` value \"{pp}\" is not a valid Universal Dependencies tag. "
+             "Non-UD tags should use the `tag` field.")
```

Comment:
If this error is only going to be used for `Doc.__init__`, then I think this should say `tags` instead of `tag`, something like:

```suggestion
             "Non-UD tags should use the `tags` argument.")
```

I wonder if want to use the same error instead of cryptic `KeyError` in `Token.pos_`, too, while we're at it, and then I'd have a different suggestion.

Reply:
Since we're dealing with it, I can go ahead and replace the `KeyError` too. What would you suggest?